client/components/data/query-post-revisions/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestPostRevisions } from 'state/posts/revisions/actions';
+
+class QueryPostRevisions extends Component {
+	componentWillMount() {
+		this.request( this.props );
+	}
+
+	componentWillReceiveProps( nextProps ) {
```

Comment:
Should we use `componentDidUpdate` instead to trigger side effects? Here's a gist on what should we do where in React Lifecycle methods https://gist.github.com/bvaughn/923dffb2cd9504ee440791fade8db5f9